src-js/grid/prototype.js

```diff
@@ -138,7 +138,7 @@ export function hexesInRangeFactory({ isValidHex }) {
    * @throws {Error} When no valid hex is passed.
    *
    * @example
-   * const Hex = Honeycomb.extendHex({ orientation: 'pointy' })
+   * const Hex = Honeycomb.extendHex({ orientation: 'POINTY' })
```

Comment:
You couldn't have known, but I only keep the `src-js` folder around in the `next` branch to be able to quickly see the v3 code. It will be removed when v4 is released. So there's no need to make changes in `src-js` 👍

Reply:
I just did project-wide search and replaced all the instances of both strings.